Source/MythicaEditor/Private/MythicaEditorSubsystem.cpp

```diff
@@ -1762,6 +1809,62 @@ void UMythicaEditorSubsystem::OnResultMeshData(const TArray<uint8>& FileData, in
     RequestData->ImportDirectory = ImportDirectory;
     SetJobState(RequestId, EMythicaJobState::Completed);
 
+    // @TODO: Find a way to render the thumbnails
+    // Look into FSlateShaderResource* FAssetThumbnail::GetViewportRenderTargetTexture() const, might be able to render that directly.
```

Comment:
In a previous iteration I was able to get asset thumbnails displaying in the UI. I did it from blueprint using AssetThumbnail widget.

I do seem to remember it was a bit janky to use. To make it refresh properly think I had set an invalid AssetPath into it and then set the correct one in to force it to refresh.

I think it was this deleted "Asset Generator" widget: https://github.com/MythicaAI/MythicaUnrealPlugin/pull/101